LibreNMS/Config.php

```diff
@@ -137,6 +137,41 @@ public static function get($key, $default = null)
         return Arr::get(self::$config, $key, $default);
     }
 
+    public static function getExecutable(string $key): string
+    {
+        $binary = self::get($key, $key);
+
+        if (Str::startsWith($binary, '/')) {
```

Comment:
maybe to prevent whereis calls every time in a bad situation, just check self::has($key)

Reply:
Either we try to find a binary once, and save the result forever, then the validator can complain, but user needs to update the config manually.
Or we test it each time, which causes it to auto-solve if the user installs the binary...